src/main/java/org/jabref/gui/groups/GroupDialogViewModel.java

```diff
@@ -212,8 +214,24 @@ private void setupValidation() {
                 input -> !StringUtil.isNullOrEmpty(input),
                 ValidationMessage.error(String.format("%s > %n %s",
                         Localization.lang("Free search expression"),
-                        Localization.lang("Search term is empty.")
-                )));
+                        Localization.lang("Search term is empty."))));
+
+        texGroupFilePathValidator = new FunctionBasedValidator<>(
+                texGroupFilePathProperty,
+                input -> {
+                    if (StringUtil.isBlank(input)) {
+                        return false;
+                    } else {
```

Comment:
@stefan-kolb Would review here: "Fail fast". Not endless nested if/else statements, but just exit early.

Therefore, I proposed:

```java
if (StringUtil.isBlank(input)) {
  return false;
}
```

Think, I will start another PR with this and the other code style thing and we can have the discussion there --> we need to get this fix in soon.